websites/serializers.py

```diff
@@ -321,8 +320,14 @@ class WebsiteStatusSerializer(
     WebsiteValidationMixin,
     WebsiteUrlSuggestionMixin,
 ):
+    has_site_metadata = serializers.SerializerMethodField(read_only=True)
+
     """Serializer for website status fields"""
 
+    def get_has_site_metadata(self, instance):
+        site_metadata = instance.websitecontent_set.filter(type="sitemetadata")
+        return bool(site_metadata and site_metadata[0].metadata)
+
```

Comment:
We're repeating this logic twice.

Maybe we should create a Mixin called `WebsiteStatusMixin`. Let me know if you need help with this.